bitriseio/keystore.go

```diff
@@ -0,0 +1,99 @@
+package bitriseio
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/xcode-project/serialized"
+)
+
+// UploadKeystoreParams ...
+type UploadKeystoreParams struct {
+	Alias          string `json:"alias,omitempty"`
+	Password       string `json:"password,omitempty"`
+	KeyPassword    string `json:"private_key_password,omitempty"`
+	UploadFileName string `json:"upload_file_name,omitempty"`
+	UploadFileSize int64  `json:"upload_file_size,omitempty"`
+}
+
+// UploadKeystoreURL ...
+func UploadKeystoreURL(appSlug string) string {
+	return fmt.Sprintf(AppsServiceURL+"%s/android-keystore-files", appSlug)
+}
+
+// UploadKeystoreConfirmURL ...
+func UploadKeystoreConfirmURL(appSlug, uploadSlug string) string {
+	return fmt.Sprintf("%s/%s/uploaded", UploadKeystoreURL(appSlug), uploadSlug)
+}
+
+// UploadKeystore ...
+func (s *AppsService) UploadKeystore(appSlug, pth string, params UploadKeystoreParams) error {
+	f, err := os.Open(pth)
```

Comment:
Did you forget to defer closing the file?

Reply:
god cacth